app/bundles/ReportBundle/Model/ExcelExporter.php

```diff
@@ -24,11 +24,9 @@ public function __construct(FormatterHelper $formatterHelper)
     }
 
     /**
-     * @param string $name
-     *
      * @throws \Exception
      */
-    public function export(array $reportData, $name)
+    public function export(array $reportData, string $name, string $output = 'php://output')
```

Comment:
What was the reason to make this change? Adding types to existing params is considered a BC break. The 3rd param is used for test only. I'm not a fan of modifying production code for testing purposes but I also don't see a way around it in this case. This change makes this method test-able 👍

Reply:
PHPSTAN enforces types in this change. If there is a workaround we can remove the type from `$name` parameter. Unfortunately I haven't found any way to test this export without making the output a parameter, so this is only for testing purposes.

Reply:
Yes, I've changed the type declaration. Should be good now.